deployment/armada/templates/service.yaml

```diff
@@ -6,12 +6,18 @@ metadata:
   labels:
 {{ include "armada.labels.all" . | indent 4 }}
 spec:
+  {{- if .Values.quickstart }}
```

Comment:
It might be better to call the option nodeport.

Reply:
Updated